src/templates/BlogPost/BlogPost.js

```diff
@@ -16,34 +16,54 @@ import styles from './BlogPost.module.scss';
 const BlogPost = ({ pageContext }) => {
   const {
     data,
+    recommendedPosts,
     recent: { previous, next },
   } = pageContext;
+
+  //selection of recommended posts in the current post
+  const RECOMMENDED_POSTS_LIMIT = 3;
```

Comment:
this code is unreadable at the moment. Can we move filter functionality to a function?

Reply:
fixed - https://github.com/Halo-Lab/halo-lab/pull/104